docs/docs/forms.mdx

```diff
@@ -360,74 +360,99 @@ to fully customize the validation process.
 
 If none of the predefined [Slot Mappings](./forms.mdx#slot-mappings) fit your use
 case, you can use the
-[Custom Action](./actions.mdx#custom-actions) `validate_{form_name}` to write your own
-extraction code. Rasa Open Source will trigger this function when the form is run.
+[Custom Action](./actions.mdx#custom-actions) `validate_<form_name>` to write your own
+extraction code. Rasa Open Source will trigger this action when the form is run.
 
-Make sure your custom action returns `SlotSet` events for every extracted value.
-The following example shows the implementation of a custom slot mapping which sets
-a slot based on the length of the last user message.
+If you're using the Rasa SDK we recommend you to extend the provided
+`FormValidationAction`. When using the `FormValidationAction`, two things are needed to
+extract customs slots:
```

Comment:
I think "custom slot" can lead to confusion. (It has confused me in the past.) We might want to consistently talk about "custom mapping", even though it will make some sentences more complicated...

Reply:
Sometimes I now just went with `additional slot` as the mapping focuses upon the extraction (not yet pushed)